keylime/cloud_verifier_tornado.py

```diff
@@ -471,16 +471,16 @@ def start_tornado(tornado_server, port):
     print("Tornado finished")
 
 
-def main(argv=sys.argv):
+def start(port_key,db_key,rev_port_key):
```

Comment:
This has to stay `main` or you will need a new `main` that calls this function. The reason being its an entry point. You can see CI is failing here over this: 
https://travis-ci.org/keylime/keylime/builds/598673045#L675